src/main/kotlin/org/rust/lang/core/psi/RsDefaultValueBuilder.kt

```diff
@@ -153,6 +158,8 @@ class RsDefaultValueBuilder(
             structLiteral.addAfter(psiFactory.createNewline(), structLiteral.lbrace)
         }
 
+        structLiteral.addTrailingCommaIfNeeded()
```

Comment:
I propose to make it a bit simplier - if struct is going to be multiline
```kotlin
val isMultiline = forceMultiLine || structLiteral.textContains('\n')
```
then add comma when adding last field (body of `if (addBefore == null)`)
```kotlin
if (isMultiline && fieldDecl == fieldsToAdd.last()) {
    structLiteral.addAfter(psiFactory.createComma(), added)
}
```

Reply:
Please, do not add any heuristics for trailing comma
It's very confusing
We just need to use some code style settings. As I can see, there isn't a single style across different languages so let's use someone

Reply:
I do not quite understand your proposal. Both our built-in formatter and rustfmt will add trailing comma in struct literals, so what is wrong if add it during code generation?

Reply:
Oh, I see that we trailing comma indeed used by default in other places. Ok then, let's add it as proposed in this PR
But ideally, we need an option to turn this behaviour off